python/ray/util/spark/cluster_init.py

```diff
@@ -380,83 +384,33 @@ def _init_ray_cluster(
     else:
         start_hook = RayOnSparkStartHook()
 
-    head_options = head_options or {}
-    worker_options = worker_options or {}
-
     spark = get_spark_session()
 
-    # Environment configurations within the Spark Session that dictate how many cpus
-    # and gpus to use for each submitted spark task.
-    num_spark_task_cpus = int(spark.sparkContext.getConf().get("spark.task.cpus", "1"))
-    num_spark_task_gpus = int(
-        spark.sparkContext.getConf().get("spark.task.resource.gpu.amount", "0")
-    )
+    ray_head_ip = socket.gethostbyname(get_spark_application_driver_host(spark))
+    ray_head_port = get_random_unused_port(ray_head_ip, min_port=9000, max_port=10000)
 
-    (
-        ray_worker_node_heap_mem_bytes,
-        ray_worker_node_object_store_mem_bytes,
-    ) = get_avail_mem_per_ray_worker_node(spark, object_store_memory_per_node)
+    include_dashboard = head_node_options.pop("include_dashboard", True)
```

Comment:
the default value of --include_dashboard is None in Ray which means Ray will decide whether dashboard can be started (any missing dependencies), why don't we just let Ray decide?